supersim_test.go

```diff
@@ -967,7 +967,7 @@ func TestInteropInvariantSucceedsWithDelay(t *testing.T) {
 
 func TestInteropInvariantFailsWhenDelayTimeNotPassed(t *testing.T) {
 	testSuite := createInteropTestSuite(t, config.CLIConfig{
-		InteropDelay: 5,
+		InteropDelay: 2, // 2 second delay
```

Comment:
can you explain more about why lowering the delay from 5 to 2 makes this test less flaky?

Reply:
@tremarkley the value here does not matter, just keeping it same as the previous test https://github.com/ethereum-optimism/supersim/blob/c6c140211c1477e25dc1c03c6540e2b015dc9d47/supersim_test.go#L910 . The error in tests was because of the next lines changed